BGRSCRPT/iocBoot/iocBGRSCRPT-IOC-01/runIOC.bat

```diff
@@ -2,12 +2,28 @@
 setlocal
```

Comment:
I think we also need to do a similar set of changes in this file for `BGRSCRPT_02`